src/main/java/Ui.java

```diff
@@ -0,0 +1,132 @@
+import java.util.Scanner;
+
+/**
+ * Represents the user interface that interacts with the user and acts as a view.
+ */
+public class Ui {
+    /**
+     * The input channel.
+     */
+    Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Displays a message indicating an error in loading the task list save file.
+     */
+    public void showLoadingError() {
+        String message = "Cannot find save file.\n"
+                + "Creating a new one, even if it's a bother";
+
+        say(message);
+    }
+
+    /**
+     * Displays the list of tasks given.
+     * @param tasks the list of tasks
+     */
+    public void showTasks(TaskList tasks) {
+        String message = "Here are your procrastinated tasks:\n"
+                + tasks;
+
+        say(message);
```

Comment:
Perhaps you can consider using System.lineSeparator() as a replacement for "\n" for system compatibilities!